flow/raster_cache.h

```diff
@@ -78,20 +78,39 @@ struct RasterCacheMetrics {
   size_t in_use_bytes = 0;
 
   /**
-   * The total cache entries that had images during this frame whether
-   * they were used in the frame or held memory during the frame and then
-   * were evicted after it ended.
+   * The total cache entries that had images during this frame.
    */
-  size_t total_count() const { return in_use_count + eviction_count; }
+  size_t total_count() const { return in_use_count; }
 
   /**
-   * The size of all of the cached images during this frame whether
-   * they were used in the frame or held memory during the frame and then
-   * were evicted after it ended.
+   * The size of all of the cached images during this frame.
    */
-  size_t total_bytes() const { return in_use_bytes + eviction_bytes; }
+  size_t total_bytes() const { return in_use_bytes; }
 };
 
+/**
+ * RasterCache is used to cache rasterized layers or display lists to improve
+ * performance.
+ *
+ * Life cycle of RasterCache methods:
+ * - Preroll stage
+ *   - RasterCacheItem::PrerollSetup
```

Comment:
Perhaps indent this one more level and indicate the recursion:

- Preroll stage
  - LayerTree::Preroll - for each Layer in the tree:
    - RCI::Setup ...
    - RCI::Finalize ...

Reply:
Done.